website/Makefile

```diff
@@ -38,8 +38,6 @@ build-image:
 # Use this if you have run `build-image` to use the locally built image
 # rather than our CI-generated image to test dependency changes.
 website-local:
-	@echo "==> Downloading latest Docker image..."
-	@docker pull hashicorp/nomad-website
```

Comment:
When running website-local, we ought to use the locally build image (with `build-image`) - so removed pulling latest non-local image here.